frontend/www/js/omegaup/problem/collection.ts

```diff
@@ -0,0 +1,25 @@
+import Vue from 'vue';
+import problem_collections from '../components/problem/Collection.vue';
+import { types } from '../api_types';
+import { omegaup, OmegaUp } from '../omegaup';
+import T from '../lang';
+import * as api from '../api';
+import * as ui from '../ui';
```

Comment:
```suggestion
import { OmegaUp } from '../omegaup';
```

muchas de estas cosas no se están usando, y es mejor no importarlas.